tests/core/shape_opencascade.cc

```diff
@@ -0,0 +1,76 @@
+/**
+ * @brief Tests the composite shape representation.
+ *
+ */
+
+// Lethe
+#include <core/shape.h>
+
+// Tests (with common definitions)
+#include <../tests/tests.h>
+
+void
```

Comment:
This is very good. It would be good to have similar tests to this one.
However, is this test only added when open cascade is included?